src/components/OrgPostCard/OrgPostCard.test.tsx

```diff
@@ -60,7 +60,7 @@ describe('Testing Organization Post Card', () => {
     postTitle: 'Event Info',
     postInfo: 'Time change',
     postAuthor: 'John Doe',
-    postPhoto: 'photoLink',
+    postPhoto: 'https://via.placeholder.com/200x100',
```

Comment:
We need to be using place holder images that are a part of our code base. Relying on external sources that could disappear is not a good practice.